yb_build_thirdparty_main.py

```diff
@@ -631,6 +632,7 @@ def init_flags(self):
             self.libs += ["-lc++", "-lc++abi"]
             # Build for macOS Mojave or later. See https://bit.ly/37myHbk
             self.cxx_flags.append("-mmacosx-version-min=10.14")
+            self.extrac_flags.append("-mmacosx-version-min=10.14")
```

Comment:
We are still duplicating this for no reason. Also we are missing other flags that other dependencies get through `self.compiler_flags` in crypt_blowfish. We should just make the crypt_blowfish build file accept an externally set value of CFLAGS, and append its own flags to it.

Reply:
Ah, if we put tis in compiler_flags, it will go into both make macros. I missed that common factor in the code.